tests/unit/conf/clusters/advanced.json

```diff
@@ -36,7 +64,12 @@
         "shard_level_metrics": [
           "IncomingBytes"
         ],
-        "shards": 1
+        "shards": 1,
+        "terraform_outputs": [
+          "username",
```

Comment:
I would output `arn` instead of `username`. The outputs we need for kinesis data stream are kinesis data stream name which can get it from the `arn`, `access_key_id` and `secret_key`.

Reply:
updated!